packages/terra-brand-footer/tests/jest/BrandFooter.test.jsx

```diff
@@ -12,6 +12,20 @@ describe('BrandFooter', () => {
 
```

Comment:
Can we add a jest test that adds both `links` and `sections` so we can verify the `sections` prop is honored as described?

Reply:
Done in 0939f42333450ab5a565782924d03fc9904b8564